app/Check/Consumers/Client/ClientBuilder.php

```diff
@@ -0,0 +1,90 @@
+<?php declare(strict_types = 1);
+
+namespace Pd\Monitoring\Check\Consumers\Client;
+
+class ClientBuilder
```

Comment:
Podle mě je toto spíš konfigurace. Je to value objekt, který je immutable. Pokud budu mít dva objekty, které mají stejné nastavení, tak to jsou pro mě ty stejné objekty. To je definice value objektu (identita definovaná vlastnostma) a to taky hezky sedí.

```suggestion
class Configuration
```